ivy/functional/frontends/jax/numpy/fft.py

```diff
@@ -27,3 +27,9 @@ def fftshift(x, axes=None, name=None):
     roll = ivy.roll(x, shifts, axis=axes)
 
     return roll
+
+
+@to_ivy_arrays_and_back
+def ifftn(a, s=None, axes=None, norm=None):
+    a = ivy.asarray(a, dtype=ivy.complex128)
```

Comment:
As it is mentioned in the [Docs](https://jax.readthedocs.io/en/latest/_autosummary/jax.numpy.fft.ifftn.html) no need to explicitly convert the input to complex, but input itself should be a complex type, this line can be omitted and we specify supported dtypes as complex using `with_supported_dtypes` decorator